wrapper-classes/BaseElement.js

```diff
@@ -0,0 +1,119 @@
+import { Timeouts } from '../constants/timeouts.js'
+
+class BaseElement {
+  constructor(name, selector) {
+    this.name = name
+    this.selector = selector
+  }
+
+  async getElement() {
+    return await $(this.selector)
+  }
+
+  async waitForExist() {
+    await this.getElement().waitForExist(Timeouts.SHORT_TIMEOUT)
```

Comment:
this won't work. you need to pass an object to this method. same for other wait methods below